### PR TITLE
AvielCo/issue74

### PR DESCRIPTION
**Pull request by AvielCo**
Added carousel that holds the 3 preview images.
see https://www.npmjs.com/package/react-material-ui-carousel 
for API

### CODE REVIEW

File: user/client/src/pages/Home/Home.css [hunk 1]
```diff
@@ -117,12 +117,12 @@ section.scan h3 {
 
 section.about {
   background: #046364;
-  height: 100vh;
   display: flex;
```
Comment (SoloNoah) [inline user/client/src/pages/Home/Home.css:120]: why did you remove the height?
not that it matters atm but it should he min-height so the section will hold all of the content in it

Comment (AvielCo) [reply user/client/src/pages/Home/Home.css:120]: removing the height made it responsive af

**Comment by SoloNoah** [comment]
You've been rejected, bitch! (love ya boo)
